strongbox-storage/strongbox-storage-indexing/pom.xml

```diff
@@ -0,0 +1,289 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@sbespalov : Have you fully merged from my master? There is no longer such a module. This has now all been extracted under the `strongbox-storage-maven-layout-provider` module.